lib/optimizely/index.js

```diff
@@ -74,6 +91,47 @@ Optimizely.prototype.page = function(page){
 };
 
 /**
+ * Send experiment data as a track event to other integrations
+ * This only sends active experiment data
+ *
+ * https://www.optimizely.com/docs/api#data-object
+ */
+
+Optimizely.prototype.sendRoots = function(){
+  if (!window.optimizely) return; // in case the snippet isnt on the page
+
+  var data = window.optimizely.data;
+  var activeExperiments = data.state.activeExperiments; // list of experiment Id's.
+  var allExperiments = data.experiments;
+
+  if (!data || !data.state || !allExperiments) return;
+
+  var variationNamesMap = data.state.variationNamesMap;
+  var variationIdsMap = data.state.variationIdsMap;
+
+  // for each active experiment send a track event
+  var self = this;
+  activeExperiments.forEach(function(experimentId){
+
+    var experimentObj = allExperiments[experimentId];
+    if (experimentObj){
+      var experimentName = experimentObj.name;
+      var variationName = variationNamesMap[experimentId];
+      var variationId = variationIdsMap[experimentId];
+
+      self.analytics.track(
+        'Experiment Viewed',
```

Comment:
@f2prateek I don't follow, can you explain what you mean?

These experiments should be only the active experiments (see code above), which is different than a listing of all the experiments defined for this project (is the former is what you meant?)

Pageload should indicate that an experiment has been viewed, and if we're filtering to retrieve only the active experiments I don't see a problem here.